crates/protocol/src/batch/single.rs

```diff
@@ -162,6 +162,12 @@ impl SingleBatch {
             if starts_with_2718_deposit(tx) {
                 return BatchValidity::Drop;
             }
+            // If isthmus is active and the transaction is a 7702, drop the batch.
+            if !cfg.is_isthmus_active(batch_origin.timestamp) {
```

Comment:
I think `batch_origin` is the wrong timestamp. It must be the timestamp of the batch itself.